internal/app/service.go

```diff
@@ -159,6 +161,9 @@ func (svc *Service) MakeItStop() {
 // configuration. It will also configure the webserver and start listening on
 // the specified port.
 func (svc *Service) MakeItRun() error {
+	if !handlers.NewDatabase().BootstrapHandler(svc.ctx.appCtx, nil, startup.NewStartUpTimer(svc.serviceKey), svc.dic) {
```

Comment:
If we have to do this here it should be refactored to not be a bootstrap handler.

Reply:
fair point I'll do this 😅